### ISSUE TITLE
Examples claim "WEBXR NOT AVAILABLE" but immersiveweb.dev works

### ISSUE THREAD

**Issue by omgitsraven** [mrdoob/three.js#18431]
##### Description of the problem

When either implementing the code shown on https://threejs.org/docs/index.html#manual/en/introduction/How-to-create-VR-content , or clicking any of its examples (under 'Next Steps'), the VRButton says "WEBXR NOT AVAILABLE" and links to immersiveweb.dev — however, on immersiveweb.dev , the "WebXR particles" demo works fine; that button says "ENTER VR", and if I click it, I can view the particles demo in VR.

In short:
- This gives "ENTER VR": https://toji.github.io/webxr-particles/
- This gives "WEBXR NOT AVAILABLE": https://threejs.org/examples/webxr_vr_cubes.html
- This gives "WEBXR NOT AVAILABLE": https://ravenworks.ca/temp/vrbuttonbug/

##### Three.js version

 r112

##### Browser

Only tried in Firefox

##### OS

Windows

##### Hardware Requirements (graphics card, VR Device, ...)

Oculus Rift

**Comment by Mugen87** [comment]
> Only tried in Firefox

Can you also try it with Chrome?

**Comment by mrdoob** [comment]
Firefox has not implemented WebXR yet.

**Comment by JeffesJiang** [comment]
I also encountered the same situation, I used Google Chrome and firefox reality test on my vr device at the same time, all the examples on the official website of threejs all show webxr not available, but all the examples on the immersiveweb.dev webpage are on Google and Firefox reality can be used normally. At the same time, I have also tested many a-frame framework vr projects in Google and Firefox. I also set up a threejs project myself, and followed the tutorial on the official website. It is still webxr not available. I hope someone can answer my doubts.

**Comment by steren** [comment]
If Firefox doesn't support WebXR, does it mean the immersive web samples use a webVR fallback? 

For example https://immersive-web.github.io/webxr-samples/ runs great on Firefox on Windows without having to turn any flag.

**Comment by JeffesJiang** [comment]
If you are not connected to a VR headset, you can use the chrome plug-in WebXR API Emulator 0.3.3.
If you have a headset with an Android system, you can enter VR normally by using the VR version of the Firefox browser.
The VR of the Android Google browser is the kind of mode that requires the mud mobile phone to be put into the glasses case, and the screen will become two left and right.
![image](https://user-images.githubusercontent.com/43066752/141070654-b9bcad4e-64a8-4678-b65c-72cba662b409.png)


Plugin url :  https://chrome.google.com/webstore/detail/webxr-api-emulator/mjddjgeghkdijejnciaefnkjmkafnnje

**Comment by steren** [comment]
My question was: how come the immersive web sample works on the regular Firefox browser but three.js does not. 

Is the immersive web sample using a WebVR fallback?

**Comment by Mugen87** [comment]
Yes, these example use a polyfill. It's https://github.com/immersive-web/webxr-polyfill

The `three.js` examples use no polyfill.
